app/models/collection.rb

```diff
@@ -1,11 +1,21 @@
 # frozen_string_literal: true
 
 # Generated by hyrax:models
+
 class Collection < ActiveFedora::Base
   include ::Hyrax::CollectionBehavior
-  include Ubiquity::UniversalMetadata
```

Comment:
what is the reason for removing these two lines? is the collection metadata from Ubiquity no longer needed?

Reply:
fixed this! thank you!!